docs/modules/ROOT/pages/directives.adoc

```diff
@@ -61,7 +61,7 @@ Reference: xref::type-definitions/default-values.adoc#type-definitions-default-v
 
```

Comment:
Since some of this content is being addressed already in schema-configuration/type-configuration.adoc, should it be featured here too? Maybe this should be addressed as a bigger adaptation when we can finally publish the restructure of the docs 🤔